Easydict/NewApp/Configuration/Configuration+Defaults.swift

```diff
@@ -55,7 +55,7 @@ extension Defaults.Keys {
     static let clearInput = Key<Bool>("EZConfiguration_kClearInputKey", default: true)
     static let keepPrevResultWhenEmpty = Key<Bool>("EZConfiguration_kKeepPrevResultKey", default: true)
     static let selectQueryTextWhenWindowActivate = Key<Bool>("EZConfiguration_kSelectQueryTextWhenWindowActivate", default: false)
-    static let enableBetaNewApp = Key<Bool>("EZConfiguration_kEnableBetaNewAppKey", default: false)
+    static let enableBetaNewApp = Key<Bool>("EZConfiguration_kEnableBetaNewAppKey", default: true)
```

Comment:
Since there seems to be no problem with the new SwiftUI Setting, we changed it to be enabled by default. 

In order to make it easier to use the new Swift/SwiftUI features, I plan to change the minimum supported system from macOS 11 to macOS 13 in the next version. 

After that, we no longer need to maintain the old Setting related code and move forward lightly.